### ISSUE TITLE
Github releases outdated

### ISSUE THREAD

**Issue by kosecki123** [energywebfoundation/iam-client-lib#191]
Seems like GitHub release stopped on `1.3.0` while npm version is currently at `2.2.0`

```
yarn info v1.22.10
{
  name: 'iam-client-lib',
  'dist-tags': {
    canary: '3.0.0-alpha.21',
    latest: '2.2.0'
  },
```

Also consider adding https://shields.io/ on the README.md with currently available NPM versions.

**Comment by Harasz** [comment]
Hi @kosecki123,
We will fix that soon.

**Comment by Harasz** [comment]
Fixed in #451
